{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -32,15 +35,25 @@ before_install:
   - lsb_release -a
 install:
   - pip install tox
+    {%- if cookiecutter.test_runner|lower == "pytest" %} pytest-travis-fold{% endif %}
   - virtualenv --version
   - easy_install --version
   - pip --version
   - tox --version
+{% if cookiecutter.test_runner|lower == "pytest" %}
+before_script:
+  - export TOX_TESTENV_PASSENV=TRAVIS
```

Comment:
Since pytest-travis-fold will be moved into `ton.ini`, I guess this should be better removed at all.